requests/adapters.py

```diff
@@ -484,11 +484,10 @@ def send(self, request, stream=False, timeout=None, verify=True, cert=None, prox
 
                     # Receive the response from the server
                     try:
-                        # For Python 2.7+ versions, use buffering of HTTP
-                        # responses
+                        # For Python 2.7, use buffering of HTTP responses
                         r = low_conn.getresponse(buffering=True)
                     except TypeError:
-                        # For compatibility with Python 2.6 versions and back
+                        # For Python 3.3+ versions, this is the default
                         r = low_conn.getresponse()
```

Comment:
The comments for this bit were oddly out of date; only this branch works on python3, while the above fails.